### DISCUSSION TITLE
Merging Images in Foocus for Wedding Invitation - Issue

### DISCUSSION THREAD

**Discussion by BipulRahi** [lllyasviel/Fooocus#2977] [Q&A] [answered]
### Thanks in advance for your help!

**Comment by mashb1t** [comment, marked as answer]
if you want to merge subjects of 2 images into one then I'd recommend to stitch the image together in an image tool like Photoshop / Gimp / etc. and use inpaint on everything other than the subjects OR only on the areas to be merged.
Generally you can also mix inpaint and image prompt to only faceswap in the inpainted area.
